### ISSUE TITLE
[Breadcrumb] Missing Keyboard navigation functionality

### ISSUE THREAD

**Issue by almedina-ms** [microsoft/microsoft-ui-xaml#3978]
**Describe the bug**

Some keyboard navigation has not been defined and, as such, it has not been implemented, for example, the Home/End keys may navigate to the first visible item or the last one 

Known issue from:
https://github.com/microsoft/microsoft-ui-xaml/pull/3891#discussion_r553067321

**Comment by StephenLPeters** [comment]
@almedina-ms please provide a link to the keyboard section of the spec when convenient.
